streams-designer/README.md

```diff
@@ -0,0 +1,155 @@
+# Simple load from Cloudant to Db2 in Streams Designer (Beta)
+This tutorial is an introduction on loading Cloudant documents and saving that data into Db2 within Streams Designer.
+In this tutorial you will:
+
+1. Set up IBM services Cloudant, Streaming Analytics, Watson Studio, and Db2 Warehouse on Cloud.
+1. Create a Python code operator that will load Cloudant documents from the `_changes` endpoint.
+1. Edit the output schema for the Python code operator.
+1. Create a Db2 operator and add the connection details.
+1. Map the Code node schema with the target columns in the Db2 table.
+1. Save and run the streams flow.
+
+**N.B. This is an example only intended as a _starting point_ for using IBM Streams Designer to extract data from Cloudant and insert into Db2 Warehouse on Cloud. It is _not_ intended as a drop-in replacement for the deprecated Cloudant warehouse integration. In particular this example does not create or alter Db2 tables or handle document deletion.**
+
+## Before you begin 
+
+These are the IBM Bluemix services required to run this tutorial:
+1. [Cloudant](https://console.bluemix.net/catalog/services/cloudant-nosql-db)
+1. [Db2 Warehouse on Cloud](https://console.bluemix.net/catalog/services/dashdb)
+1. [Streams Designer](https://www.ibm.com/cloud/streams-designer)
+
+# Setting up IBM Cloud services
+
+## Cloudant
+1. [Create an IBM Cloudant](../cloudant/create.md) instance.
+1. [Replicate the tutorial data](../cloudant/replicate.md).
+
+## Streaming Analytics
```

Comment:
Perhaps this and Watson Studio should be third level `###` headers under `## Streams Designer` ?